containerd/README.md

```diff
@@ -12,7 +12,33 @@ Containerd is a core Agent 6 check and thus needs to be configured in both in `d
 
 In `datadog.yaml`, configure your `cri_socket_path` for the Agent to query Containerd. In `containerd.d/conf.yaml`, configure the Check instance settings (such as `filters`) for the events.
 
-**Note**: If you are using the Agent in a container, setting the `DD_CRI_SOCKET_PATH` environment variable automatically enables the `Containerd` Check with the default configuration.
+#### Installation on containers
+
+If you are using the Agent in a container, setting the `DD_CRI_SOCKET_PATH` environment variable to the Containerd socket automatically enables the `Containerd` Check with the default configuration.
+
+For example, to install the integration on Kubernetes, edit your `datadog-agent.yaml` to map the Containerd socket from the host node to the daemonset and set the `DD_CRI_SOCKET_PATH` to the daemonset mountPath:
+
+```
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: datadog-agent
+spec:
+  template:
+    spec:
+      containers:
+        - name: datadog-agent
```

Comment:
we should probably use some `[...]` to indicate that some part of the daemonset are omitted?